mmv1/templates/terraform/examples/dns_managed_zone_quickstart.tf.erb

```diff
@@ -38,7 +38,7 @@ resource "google_compute_firewall" "default" {
 # to create a DNS zone
 resource "google_dns_managed_zone" "<%= ctx[:primary_resource_id] %>" {
   name          = "<%= ctx[:vars]['example_zone_googlecloudexample'] %>"
-  dns_name      = "googlecloudexample.com."
+  dns_name      = "googlecloudexample.net."
```

Comment:
what happened to the googlecloudexample.com DNS zone? Can we recover it rather than changing the one we're testing against?

Reply:
Riley agreed with changing the domain name. It doesn't seem easy to own domains to run tests.

Reply:
Will this test end up purchasing the DNS?

Reply:
I don't think we purchase the DNS with this test